x-pack/test/functional/page_objects/infra_hosts_view.ts

```diff
@@ -29,11 +29,11 @@ export function InfraHostsViewProvider({ getService }: FtrProviderContext) {
     },
 
     async clickSelectedHostsButton() {
-      return testSubjects.click('infraUseHostsTableButton');
+      return testSubjects.click('hostsViewTableSelectHostsFilterButton');
```

Comment:
Nit: Now I saw that we have 2 different test ids versions - `hostsView-*` (line [24](https://github.com/elastic/kibana/pull/160418/files#diff-bc49010b083e4c29571f02936e81611647ad92b79a60d0dd739df7fef7ead24aR24) for example) and the new one `hostsView*` because of the new eslint rule test ids creation, do we want to keep the old way with a `-` or we can also have them like this. 🤔

Reply:
Yes, we can do that, I think we can make them consistent, the `infraHosts...` is fine 👍 Not need to change them in this PR as this will affect other ids and we can first verify which of those are used for telemetry